acousticbrainz/views/data.py

```diff
@@ -46,22 +47,41 @@ def view_high_level(mbid):
 def summary(mbid):
     lowlevel, highlevel, genres, moods, other = get_summary_data(mbid)
 
-    info = _get_track_info(mbid, lowlevel['metadata'])
+    info = None
+
+    if lowlevel == None: # No metadata in acousticbrainz.. try to fetch data from musicbrainz
+        info = _get_track_info(mbid)
+    else: # Try to get metadata from musicbrainz with second priority as the one retrieved from acoustic
+        info = _get_track_info(mbid, lowlevel['metadata'])
```

Comment:
You can simplify this by defining call to `_get_track_info` only once.

    info = _get_track_info(mbid, lowlevel['metadata'] if lowlevel else None)